README.md

```diff
@@ -339,21 +339,28 @@ eval $(starship init ion)
 <details>
 <summary>Nushell</summary>
 
-Add the following to the end of your Nushell env file (find it by running `$nu.env-path` in Nushell):
-
-```sh
-mkdir ~/.cache/starship
-starship init nu | save -f ~/.cache/starship/init.nu
+Since Nu does [not support `eval`](https://www.nushell.sh/book/how_nushell_code_gets_run.html#eval-function) the initialization script is saved in `env.nu`:
+
+```nushell
+'
+let starship_path = $nu.default-config-dir | path join scripts starship.nu
+if $nu.is-interactive {
+  starship init nu | save $starship_path --force
+}' | save $nu.env-path --append
+'
+if $nu.is-interactive {
+  use starship.nu
+}' | save $nu.config-path --append
```

Comment:
as far as i know, this does not work :thinking: 
did it change on the latest `main` of Nushell?

Reply:
Yes you can run it directly to try 👍🏼

Reply:
Removed the conditional check based on additional feedback that it doesn't work in all cases.